{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.0",
    "cveMetadata": {
        "cveId": "CVE-2023-26302",
        "assignerOrgId": "cc1ad9ee-3454-478d-9317-d3e869d708bc",
        "state": "PUBLISHED",
        "assignerShortName": "canonical",
        "dateReserved": "2023-02-21T22:36:47.685Z",
        "datePublished": "2023-02-22T22:02:35.556Z",
        "dateUpdated": "2023-02-22T22:02:35.556Z"
    },
    "containers": {
        "cna": {
            "affected": [
                {
                    "collectionURL": "https://github.com/executablebooks/markdown-it-py/releases",
                    "defaultStatus": "unaffected",
                    "packageName": "markdown-it-py",
                    "platforms": [
                        "Linux",
                        "MacOS",
                        "Windows"
                    ],
                    "product": "markdown-it-py",
                    "repo": "https://github.com/executablebooks/markdown-it-py",
                    "vendor": "Executable Books",
                    "versions": [
                        {
                            "changes": [
                                {
                                    "at": "commit 53ca3e9",
                                    "status": "unaffected"
                                }
                            ],
                            "lessThan": "v2.2.0",
                            "status": "affected",
                            "version": "0",
                            "versionType": "semver"
                        }
                    ]
                }
            ],
            "credits": [
                {
                    "lang": "en",
                    "type": "finder",
                    "value": "Mark Esler"
                },
                {
                    "lang": "en",
                    "type": "remediation developer",
                    "value": "Chris Sewell"
                }
            ],
            "datePublic": "2023-02-22T06:39:56+01:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "Denial of service could be caused to the command line interface of markdown-it-py, before v2.2.0, if an attacker was allowed to use invalid UTF-8 characters as input.\n"
                }
            ],
            "impacts": [
                {
                    "capecId": "CAPEC-267",
                    "descriptions": [
                        {
                            "lang": "en",
                            "value": "CAPEC-267"
                        }
                    ]
                }
            ],
            "metrics": [
                {
                    "cvssV3_1": {
                        "version": "3.1",
                        "attackVector": "LOCAL",
                        "attackComplexity": "LOW",
                        "privilegesRequired": "LOW",
                        "userInteraction": "NONE",
                        "scope": "UNCHANGED",
                        "confidentialityImpact": "NONE",
                        "integrityImpact": "NONE",
                        "availabilityImpact": "LOW",
                        "baseSeverity": "LOW",
                        "baseScore": 3.3,
                        "vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L"
                    },
                    "format": "CVSS"
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-173",
                            "description": "CWE-173",
                            "lang": "en",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "timeline": [
                {
                    "lang": "en",
                    "time": "2023-02-07T14:19:00.000-06:00",
                    "value": "Reported to upstream."
                },
                {
                    "lang": "en",
                    "time": "2023-02-22T06:39:56+01:00",
                    "value": "Patched by upstream."
                }
            ],
            "references": [
                {
                    "tags": [
                        "patch"
                    ],
                    "url": "https://github.com/executablebooks/markdown-it-py/commit/53ca3e9c2b9e9b295f6abf7f4ad2730a9b70f68c"
                }
            ],
            "title": "markdown-it-py CLI crash on invalid UTF-8 characters",
            "providerMetadata": {
                "orgId": "cc1ad9ee-3454-478d-9317-d3e869d708bc",
                "shortName": "canonical",
                "dateUpdated": "2023-02-22T22:02:35.556Z"
            }
        }
    }
}